spec/features/post_index_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+RSpec.feature 'Post Index', type: :feature do
+  let(:user) { User.create(name: 'Noel', bio: 'He is a good programmer') }
+  let!(:post) { Post.create(author: user, title: "first post's title", text: 'first text') }
+  let!(:comment1) { Comment.create(author: user, post:, text: 'hello Tom') }
+  let!(:comment2) { Comment.create(author: user, post:, text: 'Thank You Tom') }
+  let!(:comment3) { Comment.create(author: user, post:, text: 'Good afternoon Tom') }
+  let!(:like1) { Like.create(user:, post:) }
+
+  before do
+    user.update(photo: 'https://unsplash.com/photos/F_-0BxGuVvo')
+  end
+
+  scenario "see user's profile picture, username, number of posts and interactions" do
+    visit user_posts_path(user)
+
+    expect(page).to have_selector('img[src="https://unsplash.com/photos/F_-0BxGuVvo"]')
+    expect(page).to have_content('Tom')
+    expect(page).to have_content('Number of posts: 0')
+    expect(page).to have_content('Comments: 3')
+    expect(page).to have_content('Likes: 1')
+  end
+end
+
+RSpec.feature 'Post Index', type: :feature do
+  let(:user) { User.create(name: 'Tom', bio: 'He is a good programmer') }
+  let!(:post) { Post.create(author: user, title: "first post's title", text: 'first text') }
+  let!(:comment1) { Comment.create(author: user, post:, text: 'hello Tom') }
+  let!(:comment2) { Comment.create(author: user, post:, text: 'Thank You Tom') }
+  let!(:comment3) { Comment.create(author: user, post:, text: 'Good afternoon Tom') }
+  let!(:like1) { Like.create(user:, post:) }
+
+  before do
+    user.update(photo: 'https://unsplash.com/photos/F_-0BxGuVvo')
+  end
+
+  scenario "see some of the post's title, body and first comments" do
+    visit user_posts_path(user)
+    expect(page).to have_content('first post\'s title')
+    expect(page).to have_content('hello Tom')
+    expect(page).to have_content("first post's title")
+  end
```

Comment:
- [ ] It keeps getting closer! I noticed here that you're testing for the title of the post twice at lines `40` and `42` I'd suggest you replace one of them to test for **the body of the post** instead as you don't have one currently.